client/hosting/overview/components/migration-overview.tsx

```diff
@@ -75,46 +80,54 @@ const MigrationOverview = ( { site }: { site: SiteDetails } ) => {
 				);
 				break;
 			default:
-				continueMigrationUrl = addQueryArgs(
-					baseQueryArgs,
-					'/setup/site-migration/site-migration-how-to-migrate'
-				);
+				return addQueryArgs( baseQueryArgs, '/setup/site-migration/site-migration-how-to-migrate' );
 		}
 	} else {
 		// For the /setup/migration, where the checkout is before the choice.
 		switch ( migrationType ) {
 			case 'diy':
-				continueMigrationUrl = addQueryArgs(
-					baseQueryArgs,
-					'/setup/migration/site-migration-instructions'
-				);
+				return addQueryArgs( baseQueryArgs, '/setup/migration/site-migration-instructions' );
 				break;
 			case 'difm':
-				continueMigrationUrl = addQueryArgs(
-					baseQueryArgs,
-					'/setup/migration/site-migration-credentials'
-				);
+				return addQueryArgs( baseQueryArgs, '/setup/migration/site-migration-credentials' );
 				break;
 			default:
-				continueMigrationUrl = addQueryArgs(
-					baseQueryArgs,
-					'/setup/migration/migration-how-to-migrate'
-				);
+				return addQueryArgs( baseQueryArgs, '/setup/migration/migration-how-to-migrate' );
 		}
 	}
+};
+
+const MigrationOverview = ( { site }: { site: SiteDetails } ) => {
+	const migrationType = getMigrationType( site );
+	const migrationStatus = getMigrationStatus( site );
+	const continueMigrationUrl = getContinueMigrationUrl( site, migrationType, migrationStatus );
+	const isPending = 'pending' === migrationStatus;
+
+	const title = isPending
+		? translate( 'Your WordPress site is ready to be migrated' )
+		: translate( 'Your migration is underway' );
+
+	const paragraph = isPending
+		? translate( 'Start your migration today and get ready for unmatched WordPress hosting.' )
+		: translate(
+				'Sit back as {{strong}}Bold Apps{{/strong}} transfers to its new home. Get ready for unmatched WordPress hosting.',
```

Comment:
I think **Bold Apps** in the design means the site's title 🤔 Probably it should be dynamically populated instead of statically having "Bold Apps' hardcoded

<img width="1233" alt="image" src="https://github.com/user-attachments/assets/49998986-905d-4196-a3dc-e3dd3b92f4e7">

cc: @fditrapani for confirmation

Reply:
Good catch. That's right @Imran92. It should be the site title, favicon, and url of the site.

Reply:
Oh no! How I didn't notice that? 🤦‍♂️ haha
Good that @Imran92 noticed that! 🙌

Reply:
I don't think it's possible, but since the type says that it's possible, I added a fallback in case the site doesn't have a name: https://github.com/Automattic/wp-calypso/pull/95695/commits/a0523cf36742e7df26e31292f78616352d4acc1d